packages/block-editor/src/hooks/align.js

```diff
@@ -59,7 +59,15 @@ export function getValidAlignments(
 ) {
 	let validAlignments;
 	if ( Array.isArray( blockAlign ) ) {
-		validAlignments = blockAlign;
+		validAlignments = ALL_ALIGNMENTS.filter( ( value ) =>
+			blockAlign.includes( value )
+		);
+		// Exists solely for backward compatibility to cover custom values.
```

Comment:
I have no idea we need it at all, but it's always better to have it covered. I'm happy to remove it if you disagree.

Reply:
Oh, it never occurred to me that we allowed _custom_ align values. I've taken a look at the code and it looks like we render them and work (add the class `has-text-align-<your-value>`) but this seems an oversight on our side, given that the control can't have an icon or be translated (see [BLOCK_ALIGNMENT_CONTROLS](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/block-alignment-toolbar/index.js#L80)).

And what happens when you add a custom value:

![Screenshot from 2020-10-19 16-41-05](https://user-images.githubusercontent.com/583546/96466026-f77c2700-1229-11eb-89cf-563735189d27.png)

I'd think it's safe to remove the code that processes custom values.

Reply:
Yes, I should do some testing myself 😅 Good point 👍

Reply:
Removed. PR rebased.